Chisel/Makefile

```diff
@@ -0,0 +1,14 @@
+PREFIX ?= /usr/local/lib
+TEMPORARY_FOLDER ?= /tmp/Chisel.dst
+
+install:
+	xcodebuild \
+		-scheme Chisel \
+		-configuration Release \
+		-sdk iphonesimulator \
+		install \
+		DSTROOT="$(TEMPORARY_FOLDER)" \
+		INSTALL_PATH=/ \
+		SKIP_INSTALL=NO
+	install -d "$(PREFIX)"
```

Comment:
Does this do anything beyond `mkdir -p`?

Reply:
Removed with above changes